### DISCUSSION TITLE
Not able to add peers on Private Multi-Node setup

### DISCUSSION THREAD

**Discussion by ghost** [gochain-io/docs#147] [Q&A]
I am trying to set up a 3-node GoChain Private node, I am facing issues while trying to connect the nodes using the admin.addPeer("enode") command. I am following the gochain docs (https://github.com/gochain/docs/tree/master/private-networks/custom) to set up the multinode setup.

Here is what I have tried,

01. Created a new account for the 3 nodes (node1, node2, node3)
02. Initialized the genesis file (node1, node2, node3)
03. Command Used to start the node : 

`gochain --networkid 2345435345 --port 30303 --config /gochain/config.toml --datadir /gochain/node --nodiscover --cache ${GOCHAIN_CACHE:-1024} --mine --unlock ${GOCHAIN_ACCT} --password /gochain/password.txt --verbosity 5`

04. Started the next 2 nodes similarly with the same networkId, different ports, and data directory.
05. Added the node2 and node3 as signers on node1
06. Tried using admin.addPeer('enode of node2') from node 1, The output that I got was true, but once checking admin.peer, it is returning an empty array.

Please check the [gist ](https://gist.github.com/arun-rs-collab/cc8128ca84372551117ec092259fbaea)for details on the genesis file and node1 info.

**Comment by treeder** [comment]
Hi @arun-rs-collab , were you able to resolve this issue? Regret the delayed response.
